sdk/advisor/Azure.ResourceManager.Advisor/ci.mgmt.yml

```diff
@@ -0,0 +1,13 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Why do we have two ci.mgmt.yml files in this RP?

Reply:
fixed